### PR TITLE
Update MIT to allow for "materials"

### PR DESCRIPTION
**Pull request by jlovejoy**
This replaces #2496 

Signed-off-by: Jilayne Lovejoy

### CODE REVIEW

File: test/simpleTestForGenerator/MIT.txt [hunk 1]
```diff
@@ -2,8 +2,17 @@ MIT License
...
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+The above copyright notice and this permission notice shall be included in all copies or substantial 
+portions of the Software.
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT 
+LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO 
+EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER 
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE 
+USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (xsuchy) [inline test/simpleTestForGenerator/MIT.txt:18]: Nitpick - please do not rewrap the tests if there is no change. I had to go word by word during a review if something changed here. 
FTR - no change here, just re-wrap.

**Comment by xsuchy** [comment]
This solves the "Material" part of https://github.com/spdx/license-list-XML/issues/2017 but does not resolve the central part of the license:

```
MODIFICATIONS TO THIS FILE MAY MEAN IT NO LONGER ACCURATELY REFLECTS KHRONOS
STANDARDS. THE UNMODIFIED, NORMATIVE VERSIONS OF KHRONOS SPECIFICATIONS AND
HEADER INFORMATION ARE LOCATED AT https://www.khronos.org/registry/
```
What should we do about it?

**Comment by xsuchy** [comment]
The original request was to accomodate https://github.com/KhronosGroup/glslang/blob/main/SPIRV/spirv.hpp (or https://github.com/KhronosGroup/SPIRV-Cross/blob/main/LICENSES/LicenseRef-KhronosFreeUse.txt it is the same text)
As noted at https://github.com/spdx/license-list-XML/issues/2017#issue-1774776649
If I understand it correctly, it still does not match any id.

**Comment by swinslow** [comment]
@xsuchy Both of those links match [`MIT-Khronos-old`](https://spdx.org/licenses/MIT-Khronos-old.html), don't they? I just ran a redline of them and they appear to be identical matches, unless I'm missing something?

**Comment by xsuchy** [comment]
Right.
So +1 from me.

**Pull request merged**